### PR TITLE
Add part of writeups for affinity CTF

### PR DESCRIPTION
**Pull request by aenniw**
Signed-off-by: Martin Mihálek <aenniw@gmail.com>

### CODE REVIEW

File: affinityctf.com/f-word/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+#### Solution:
+
+- bruteforce
+
+```bash
+function bfs() {
+    echo '^^^^!^^^^*-A?*--&-----&&^^^&*^!^^^*-A?*^&!^^^^^*-A?*--&*^^!^^*-----A?*&-!^^^^^*-A?*^^&^!^^^^^*-A?*&*^!^^^*-A?*^^&!^^^*-A?*---&-!^*---A?*&^!^*------A?*&---!^*--A?*&--!^*-----A?*&^^!^^^*--A?*&^^^^^^^^^^^^&^^^&*^!^^^*-A?*^^^&-------------&-!^^^*--A?*--&^^^!^^^^^*-A?*^&---!^*---A?*--&--------&-----&^!^*---A?*^&^!^^^*--A?*^&^!^^^^^*--A?*^&^!--*^^^A?*^&!^^^^^*-A?*--&^^^^^^^^^^^^^^&^^^^^&-!^*---A?*-&&&*^^!^^*---A?*&'
+}
+
+for s1 in '+' '-' '>' '<' ']' '.' ; do
```
Comment (fabrymarko) [inline affinityctf.com/f-word/README.md]: where the brainf*ck did you find this?

File: affinityctf.com/man-in-the-middle/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#### Challenge:
+
+Note: put flag into AFFCTF{} format. [Man_In_The_Middle.pcap.gz](./Man_In_The_Middle.pcap.gz ":ignore")
+
+---
+
+#### Solution:
+
+- confidential data can be found in FTP dir listing and later on can be captured binary see: ![payload.png](./payload.png ":ignore")
+- password can be found in SMTP `Horse Battery Staple Correct` see: ![mail.png](./mail.png ":ignore")
```
Comment (fabrymarko) [inline affinityctf.com/man-in-the-middle/README.md]: https://xkcd.com/936/

Comment (aenniw) [reply affinityctf.com/man-in-the-middle/README.md]: lol

File: affinityctf.com/f-word/README.md (hunk 1 above)
Comment (aenniw) [reply affinityctf.com/f-word/README.md]: wrote it :laughing:

**Pull request merged**